lib/vector/vedit/zbulk.c

```diff
@@ -118,17 +119,19 @@ int Vedit_bulk_labeling(struct Map_info *Map, struct ilist *List, double x1,
         }
 
         if (Vect_rewrite_line(Map, line, type, Points, Cats) < 0) {
-            return -1;
+            nlines_modified = -1;
+            goto free_exit;
         }
         nlines_modified++;
 
         value += step;
     }
 
     if (Vect_delete_line(Map, temp_line) < 0) {
-        return -1;
+        nlines_modified = -1;
```

Comment:
should this also have a goto like all other parts of the function, in prevision of future code changes that adds code underneath? (to prevent semantic conflicts in the future)

Reply:
@nilason That was the only remaining question I had that prevented to merge this right away.

Reply:
Sorry, overlooked this. I think there’s no need to `goto` to next line, I’m not particularly worried in this case with respect to future changes, let’s keep it simple.

Reply:
Perfect then!